suite/tests/libutil/frontend_test.c

```diff
@@ -53,11 +58,22 @@ main()
         return -1;
     }
 
-    res = drfront_remove_dir("test_dir");
+    path = getenv("PATH");
+    strncpy(path_buf, "test_dir:", 9);
+    strncat(path_buf, path, 512 - 10);
```

Comment:
nit: Better to use `sizeof(path_buf)` instead of repeating the 512 so they'll stay in sync; even better to use our BUFFER_SIZE_ELEMENTS macro, but that may not be available w/ the current headers here.

Reply:
Well I tried to tweak this and get the test to run on Windows but there are still Windows and Mac issues and I have no more time.